src/fragments/forms/map-form/i18n/map-form.i18n.cs-cz.js

```diff
@@ -0,0 +1,9 @@
+
+export default {
+  mapForm: {
+    placesAndDirections: 'Vyhledat a přejít',
+    isochrones: 'Dosah',
+    uploadedContentRendered: 'Zobrazení nahraného obsahu',
```

Comment:
Automatic translation suggests "View uploaded content" which is different from the original "Uploaded content rendered" - is this an issue with the automatic translation?
The context here is to inform the user that their uploaded content has finished rendering, and "View uploaded content" sounds like something the user could click on to view their content…